src/Compiler/SyntaxTree/LexFilter.fs

```diff
@@ -1201,6 +1201,10 @@ type LexFilterImpl (
                             delayToken (pool.UseShiftedLocation(tokenTup, INFIX_AT_HAT_OP "@", 1, 0))
                             delayToken (pool.UseShiftedLocation(tokenTup, LESS res, 0, -1))
                             pool.Return tokenTup
+                        | GREATER_BAR_RBRACE ->
```

Comment:
Thanks @edgarfgp !

Just like with the other change, I think this PR should also be covered by a languageversion check, to avoid mismatches between different compiler versions.


If I got the change right, having a `when langversion ...` at this match clause might do it?
(not sure if there is a better place and how well that will recover)

Reply:
@T-Gro Updated as requested . Arguably this is a bugfix rather than a feature. But I guess there is not other option to prevent versions mismatches.